docs/modules/migrate/pages/rolling-restart.adoc

```diff
@@ -0,0 +1,45 @@
+= Upgrade Cluster to {enterprise-product-name}
+:description: You can use an in-place rolling restart to move each member on your Open Source cluster to an Enterprise cluster.
+
+{description}
+
+This approach ensures that you can move your cluster without interruption to the service. 
+Similar to the rolling upgrade process, a rolling restart allows you to move one member at a time.
+
+The codebase of your {open-source-product-name} cluster must match the target {enterprise-product-name} release. 
+Currently, rolling restart can only be used to update your 5.5 {open-source-product-name} cluster members.
+
+== Best Practice
+
+Before starting a rolling restart, consider the following best practices:
+
+* Backup your IMap and ReplicatedMap data structures. This is not essential, but it is recommended practice
+* If the {open-source-product-name} member is a different version to the target {enterprise-product-name} member, update the codebase to match the target {enterprise-product-name} release
```

Comment:
Wouldn't this defeat the whole purpose of the OS to EE restart?

Say (in the future) user is running 5.5.x OS cluster, and they want to upgrade to 5.6.x EE.

This says to update OS cluster to 5.5 first to 5.6, and then do OS to EE restart.
But they can't update the cluster without losing the data.

What they need to do is OS to EE restart first on 5.5 and the do RU from 5.5 to 5.6.

Reply:
Possibly my misunderstanding (and related to the comment above over 5.5 v current version, I understood that the OS members need to be 5.5 before they can be moved.

If I was incorrect, this bullet can be removed. Otherwise, we need to clarify somehow that the OS must be on 5.5; and that statement would need to be removed for the next release.